src/vm/util.cpp

```diff
@@ -2864,6 +2864,21 @@ void DACNotify::DoJITNotification(MethodDesc *MethodDescPtr)
     DACNotifyExceptionHelper(Args, 2);
 }
 
+void DACNotify::DoJITNotification2(MethodDesc *MethodDescPtr, TADDR NativeCodeLocation)
```

Comment:
DoJITNotification above is dead code, remove it and then rename this one to not have a '2'.